submissions/description/sensitive_data_exposure/sensitive_token_in_url/user_facing/template.md

```diff
@@ -0,0 +1,20 @@
+# Sensitive Token Exposed in URL User Facing
+
+## Overview of the Vulnerability
+
+Sensitive data can be exposed when it is not behind an authorization barrier. When this information is exposed it can place the application at further risk of compromise. The application discloses a sensitive token in the URL that is user facing which can be captured by an attacker, allowing them to locally reset a legitimate user’s account password to one they control, successfully taking over the user’s account.
```

Comment:
Wording change required as this is not entirely accurate

> Sensitive data can be exposed when it is not behind an authorization barrier. When this information is exposed it can place the application at further risk of compromise. The application discloses a sensitive token in the URL that is user facing which can be captured by an attacker. This allows the attacker to gain access to a legitimate user’s account, breaching the confidentiality and integrity of their account.